docs/director/fetch-requests.md

```diff
@@ -0,0 +1,20 @@
+# Fetch Requests
+
+Packages API supports providing documentation for API Definitions, Event Definitions. User can provide the documentation
+during the call to Director or use a Fetch Request. If a Fetch Request is specified Director will call the specified URL
+and download the specification.
```

Comment:
```suggestion
and downloads the specification.
```

Reply:
I still don't know what Fetch Request is. Is it a component? A feature? How do I use the Fetch Request?